resources/js/features/achievements/components/AchievementGroup/AchievementGroup.tsx

```diff
@@ -0,0 +1,23 @@
+import type { FC } from 'react';
+
+import { UnlockableAchievementAvatar } from '@/features/achievements/components/UnlockableAchievementAvatar/UnlockableAchievementAvatar';
+
+interface AchievementGroupProps {
+  group: App.Community.Data.AchievementGroup;
+  showGame?: boolean;
+}
+
+export const AchievementGroup: FC<AchievementGroupProps> = ({ group, showGame = false }) => {
+  return (
+    <div className="mb-0 mt-4">
```

Comment:
Better to remove this `className` prop and lift these layout concerns up to the parent. Generally the higher you can lift a layout concern, the better it is for maintainability and component reusability later.

In _achievement-checklist.tsx_:
```tsx
// "flex flex-col" means "stack these child nodes on top of each other".
// "gap-*" is how much space between each thing there should be.
<div className="flex flex-col gap-4">
  {groups.map((group, index) => (
    <AchievementGroup group={group} showGame={true} key={`ach-group-${index}`} />
  ))}
</div>
```

Reply:
Done.